appscale/tools/templates/AppScalefile-cloud

```diff
@@ -1,13 +1,16 @@
 ---
 # The underlying cloud infrastructure to deploy AppScale over.
 # Set this to 'ec2' if running on Amazon EC2, 'euca' if running on Eucalyptus,
-# or 'gce' if running on Google Compute Engine.
+# 'gce' if running on Google Compute Engine, or 'azure' if running on
+# Microsoft Azure.
 infrastructure : 'ec2'
 
-# The Amazon or Eucalyptus Machine Image (ami or emi) that has AppScale
-# installed on it. When running on Google Compute Engine, set this to the name
-# of the image you set when you ran 'gcutil addimage'.
+# The Amazon / Eucalyptus Machine Image (ami or emi) or Azure image resource
+# url that has AppScale installed on it. When running on Google Compute Engine,
+# set this to the name of the image you set when you ran 'gcutil addimage'.
 machine : 'ami-XXXXXX'
+# machine: 'https://{storageaccount}.blob.core.windows.net/system/' +
```

Comment:
Is this the easiest way of providing the image? Can you get this from their web console easily?
Alternatively, should we be asking for the three parameters and then constructing the url internally?